lumen/ai/ui.py

```diff
@@ -319,12 +319,21 @@ def __init__(
         if levels.get(self.log_level) < 20:
             self.interface.callback_exception = "verbose"
         self.interface.disabled = True
-        self._verbose_toggle = Switch(
-            label="Verbose",
-            value=False,
-            color="light",
-            margin=(18, 10, 10, 15),
-            size="small"
+        # Create consolidated settings menu toggle
+        self._settings_menu = MenuToggle(
+            label="Toggle Settings",
+            icon="settings",
+            color="primary",
+            margin=(0, 5, 0, 5),
+            align="center",
+            persistent=True,
+            on_click=self._handle_settings_click,
+            sx={
+                'borderRadius': '6px',  # Less rounded corners
+                'boxShadow': '0px 3px 5px -1px rgba(0,0,0,0.2)',  # Matching shadow
+                'width': '200px',  # Consistent width
+                'fontSize': '13px'  # Consistent font size
```

Comment:
fontSize should be expressed in em so it scales if needed. Ideally we'd express boxShadow via the Mui shadow CSS variables and width should be set on the component itself.